core/src/main/kotlin/org/evomaster/core/problem/rest/builder/RestActionBuilderV3.kt

```diff
@@ -621,16 +624,14 @@ object RestActionBuilderV3 {
         }
 
         when (p.`in`) {
-
-            "query" -> {
-                params.add(QueryParam(name, gene, p.explode ?: true, p.style ?: Parameter.StyleEnum.FORM))
-            }
+            "query" -> params.add(QueryParam(name, gene, p.explode ?: true, p.style ?: Parameter.StyleEnum.FORM)
+                    .apply { this.description = description })
             /*
                 a path is inside a Disruptive Gene, because there are cases in which we want to prevent
                 mutation. Note that 1.0 means can always be mutated
              */
             "path" -> params.add(PathParam(name, CustomMutationRateGene("d_", gene, 1.0)))
-            "header" -> params.add(HeaderParam(name, gene))
+            "header" -> params.add(HeaderParam(name, gene).apply { this.description = description })
```

Comment:
shouldn't this apply to all kinds of `param`?  so could have the `apply` outside of the `when`

Reply:
We are directly add the `Param` to the `MutableList`. So I added the description directly to other types also.